code/modules/power/cable.dm

```diff
@@ -431,32 +436,31 @@
 	/// cable_surr uses the unique ordinal dirs to save directions as it needs to store up to 8 at once
 	var/cable_surr = 0
 
+	/// reinforced, thick cables. They should also connect to the regular kind.
+	node
```

Comment:
please actually path these instead of relative pathing

Reply:
i've now pathed the node spawners, reinforced spawners and reinforced node spawners